tests/phpunit/tests/http/http.php

```diff
@@ -9,6 +9,9 @@ class Tests_HTTP_HTTP extends WP_UnitTestCase {
 	const FULL_TEST_URL = 'http://username:password@host.name:9090/path?arg1=value1&arg2=value2#anchor';
 
 	/**
+	 * @ticket 24976
```

Comment:
```suggestion
```

Incorrect ticket ref, [WP#24976](https://core.trac.wordpress.org/ticket/24976) was for the creation of the wordpress-develop repo (combined the tests and src repos in to one).

Reply:
This was to note when the test method was first introduced. I couldn't get an earlier ticket reference, and just including `@ticket 56231` would give the false impression that it was only added with reference to adding fragment support.

Reply:
Added `@since 3.7 Test method introduced.` in [d528ab7](https://github.com/WordPress/wordpress-develop/pull/2983/commits/d528ab7f9870025016893e0275a2926ac3a3d1fd).